src/index.ts

```diff
@@ -1,8 +1,23 @@
 import { createServer } from '@/util/server';
+import Database, { connectDatabase } from '@/database';
 
-const app = createServer();
-const port = process.env.PORT ?? 8080;
+export let db: Database;
 
-app.listen(port, () => {
-  console.log(`Server is running at https://localhost:${port}`);
-});
+async function main() {
+  const app = createServer();
+  const port = process.env.PORT ?? 8080;
```

Comment:
i think is `||`

Reply:
sorry I made a stupid problem

Reply:
> i think is `||`

I'm not sure wdym. It's an operator in TypeScript that returns the default value (rhs) if the lhs expression is null or undefined. `||` will return the rhs if the lhs is a falsy value, including `false`, `0`, empty string, etc.